src/components/implementation/no_interface/vkernel/vk_types.h

```diff
@@ -12,4 +12,7 @@ enum vm_captbl_layout {
 #define VM_COUNT        2	/* virtual machine count */
 #define VM_UNTYPED_SIZE (1<<26) /* untyped memory per vm = 64MB */
 
+#define VM_BUDGET_FIXED 400000
```

Comment:
Yes, it is small. On the HW we're using here, its much lesser: ~117us. This is not going to remain for long, especially after I change the scheduling policy to something like credit-based, I'll change it accordingly.